GSA/made-in-america
# [MIA-390] Implement new Federal Financial Assistance waiver page design

## Description
- This PR implements the Federal Financial Assistance page. 
- Removed sidenav

## How to test

- pull down branch and and navigate via the drop down.
- Click on one or two links to make sure the open in a new tab.

To prepare for code review, please follow the following checks.
